protocols/797dee-normalization/README.md

```diff
@@ -0,0 +1,74 @@
+# Normalization
```

Comment:
We should change the title of this to not get confused with the verified version of the Normalization protocol (or add .hide-from-search)

Reply:
Or update the normalization protocol